cdl/dtd_service.go

```diff
@@ -0,0 +1,107 @@
+package cdl
+
+import (
+	"fmt"
+	"github.com/go-playground/validator/v10"
+	"io"
+	"net/http"
+	"path"
+)
+
+type DataTypeDefinition struct {
```

Comment:
Please run `go fmt` and commit

Reply:
"go fmt " command has been run, the code in dtd_service.go, dtd_service_test.go and cdl/client.go have been formatted.